libraries/AP_ADSB/AP_ADSB.h

```diff
@@ -53,6 +55,12 @@ class AP_ADSB {
         return _singleton;
     }
 
+    // ADSB driver types
+    enum class Type {
+        None   = 0,
```

Comment:
```suggestion
        None                  = 0,
```

(enough to make them align :-) )